sdk/tvOS/OoyalaTVSkinSDK/OOTVGestureManager.m

```diff
@@ -86,6 +86,13 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldBeRequi
   return NO;
 }
 
+- (BOOL)gestureRecognizerShouldBegin:(UIGestureRecognizer *)gestureRecognizer {
+  if ((gestureRecognizer == self.panGesture) && self.controller.player.isPlaying) {
+    return NO;
+  }
+  return YES;
```

Comment:
we return yes by default, only return no for pan gesture when isPlaying is true.